src/required_resources.h

```diff
@@ -116,6 +116,9 @@ class RequiredResources
   std::vector<Probe> special_probes;
   std::vector<Probe> watchpoint_probes;
 
+  // List of probes using userspace symbol resolution
+  std::unordered_set<std::string> probes_using_usym;
```

Comment:
```suggestion
  std::unordered_set<std::reference_wrapper<Probe>> probes_using_usym;
```
Storing a reference will be a bit faster and use less memory than basing this set on strings. Theoretically we could also get probe name collisions, which references would prevent.